WordPressAuthenticator/Unified Auth/StoredCredentialsAuthenticator.swift

```diff
@@ -61,11 +86,9 @@ class StoredCredentialsAuthenticator: NSObject {
             // No-op for now, but we can decide to implement AppleID login through this authenticator
             // by implementing the logic here.
             break
-        case _ as ASPasswordCredential:
-            // TODO: No-op for now.  The code below will be enabled in my next PR.
-            //
-            //let loginFields = LoginFields.makeForWPCom(username: credential.user, password: credential.password)
-            //loginFacade.signIn(with: loginFields)
+        case let credential as ASPasswordCredential:
```

Comment:
nitpick: In this case it should be plural, `credentials`, since it stores the username and the password.

Reply:
The credential is the username and password combination - we're not dealing with multiple credentials.

Apple names the types `ASAuthorizationAppleIDCredential` and `ASPasswordCredential` in singular form - this is why I followed their lead (and I agree).